testing/tests/DevExpress.performance/overlaysStyleRecalculations.tests.js

```diff
@@ -77,4 +77,52 @@ QUnit.performanceTest('dxOverlay should not force relayout on creation', functio
 
         assert.measureStyleRecalculation(measureFunction, shading ? 18 : 17);
     });
+
+    [true, false].forEach(enableBodyScroll => {
+        QUnit.performanceTest(`showing dxPopup with shading=${shading} & enableBodyScroll=${enableBodyScroll} should be shown with minimum count of relayouts`, function(assert) {
```

Comment:
Update in all new tests plz
```suggestion
        QUnit.performanceTest(`Popup with shading=${shading} & enableBodyScroll=${enableBodyScroll} should be shown with minimum count of relayouts`, function(assert) {
```

Reply:
I will

Reply:
https://github.com/DevExpress/DevExtreme/pull/23926/commits/fd2b6b773237f86f0fd2b04d5786a12da38224c0